doc/rules/control_structure/no_empty_block.rst

```diff
@@ -0,0 +1,49 @@
+=======================
+Rule ``no_empty_block``
+=======================
+
+There must not be any empty blocks.
+
+Warning
+-------
+
+Using this rule is risky
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Risky if the block has side effects.
+
+Examples
+--------
+
+Example #1
+~~~~~~~~~~
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+   -<?php if ($foo) {}
+   +<?php 
+   \ No newline at end of file
+
+Example #2
+~~~~~~~~~~
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+   -<?php switch ($foo) {}
+   +<?php 
+   \ No newline at end of file
+
+Example #3
+~~~~~~~~~~
+
+.. code-block:: diff
+
+   --- Original
+   +++ New
+   -<?php while ($foo) {}
```

Comment:
`while($itrator->consume()) {}`

I wonder if it makes sense to have config which structures could be removed

Reply:
It can be added later as improvement, I would rather leave it as is now.